apps/web/src/ee/clerk/components/QuestionnaireForm.tsx

```diff
@@ -45,19 +47,16 @@ export function QuestionnaireForm() {
   const location = useLocation();
   const [_, setParams] = useSearchParams();
 
-  const { mutateAsync: updateOrganizationMutation } = useMutation<{ _id: string }, IResponseError, any>(
```

Comment:
@ChmaraX the `useMutation` hook felt unnecessary at this point. Could you please confirm that's correct?